src/package/packageVersion.ts

```diff
@@ -277,6 +349,26 @@ export class PackageVersion {
     return this.data.SubscriberPackageVersionId;
   }
 
+  public async getPackageId(): Promise<string> {
+    if (!this.data.Package2Id) {
+      await this.getPackageVersionData();
+    }
+    return this.data.Package2Id;
+  }
+
+  public async getPackageType(): Promise<PackageType> {
```

Comment:
`getPackageId` and `getPackageType` should have jsDoc.  `getPackageId` jsDoc should mention the returned ID prefix (0Ho).